api/cache.go

```diff
@@ -22,8 +22,9 @@ type (
 	// userTierCacheEntry allows us to cache some basic information about the
 	// user, so we don't need to hit the DB to fetch data that rarely changes.
 	userTierCacheEntry struct {
-		Tier      int
-		ExpiresAt time.Time
+		Tier          int
+		QuotaExceeded bool
```

Comment:
It's 3 items. I find it much more important to present what we are trying to do with the struct by starting with the most important fields than to make it easier for people to find a given field. This will change when the number of fields grows but at this stage alphabetizing would do more harm to readability than help.